config/toggles.js

```diff
@@ -179,6 +179,10 @@ const shortTermToggles = {
   requireHelpEmailOrHelpUrl: environmentVariableTrue(
     pe, 'REQUIRE_HELP_EMAIL_OR_HELP_URL'
   ),
+
+  autoDeactivateRooms: environmentVariableTrue(
```

Comment:
Create a user story to follow up on this short-term toggle to either remove it or make it a long-term toggle?

Reply:
I have created the story to remove the short term toggle